# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1260761724 28800
#      Sun Dec 13 19:35:24 2009 -0800
# Node ID 13a85d3e13bbe557952990bad01c13eb8fe41da0
# Parent  711aa22ff83dbea3198e4c42fffe6119ed23166e
Fix typo and distribute new ppder.m script

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2009-12-09  Rik <octave@nomad.inbox5.com>
+
+	* polynomial/module.mk: Correct typo and distribute newly created ppder.m
+
 2009-12-13  Jaroslav Hajek  <highegg@gmail.com>
 
 	* ppder.m: New function.
 	* ppint.m: New function.
 	* ppjumps.m: New function.
 
 2009-12-09  Rik <octave@nomad.inbox5.com>
 
diff --git a/scripts/polynomial/module.mk b/scripts/polynomial/module.mk
--- a/scripts/polynomial/module.mk
+++ b/scripts/polynomial/module.mk
@@ -15,17 +15,17 @@ polynomial_FCN_FILES = \
   polynomial/polyfit.m \
   polynomial/polygcd.m \
   polynomial/polyint.m \
   polynomial/polyout.m \
   polynomial/polyreduce.m \
   polynomial/polyval.m \
   polynomial/polyvalm.m \
   polynomial/ppval.m \
-  polynomial/ppval.m \
+  polynomial/ppder.m \
   polynomial/ppint.m \
   polynomial/ppjumps.m \
   polynomial/residue.m \
   polynomial/roots.m \
   polynomial/spline.m \
   polynomial/unmkpp.m
 
 FCN_FILES += $(polynomial_FCN_FILES)
